SchemaStore/schemastore
# Add vtestunit schema v2.0.0

- add version v2.0.0 to vtestunit schema
- add positive and negative tests for new v2.0.0 features
- add $id property to vtestunit schema (see #3242)
- improve description of vtestunit schema
